### DISCUSSION TITLE
Open Shell or Classic Shell

### DISCUSSION THREAD

**Discussion by darkrats** [Open-Shell/Open-Shell-Menu#1698] [Q&A]
I'm unsure whether to go with Open Shell or just use the last version of Classic Shell. Is there any place I can go to see a list of changes or added functions since Open Shell took over the reins of this product? I'm using Windows 10, if that's something you need top know.

**Comment by ge0rdi** [comment]
You can see changes that were done in [releases](https://github.com/Open-Shell/Open-Shell-Menu/releases) page.
There was quite big number of fixes/improvements done during last 5 years, so there is no short list of them.

Ofc, you can use whatever version (even original `Classic-Shell`) that works for you.
